### PR TITLE
Add: Upgrade Voice For MOAB, Sentry Drone Gun, Neutron Shells And Fortified Structure

### PR DESCRIPTION
**Pull request by commy2**
Files are already from the game, but can for this use case only be read from this folder, so they have to be copied there unfortunately.

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Upgrade.ini [hunk 1]
```diff
@@ -172,6 +172,7 @@ Upgrade Upgrade_AmericaMOAB
   BuildTime          = 60.0
   BuildCost          = 4000
   ButtonImage        = SAMOAB
+  ResearchSound      = EvaVoiceUpgradeMOAB ; @tweak commy2 20/08/2022 Add voice line.
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Upgrade.ini]: Patch104p keyword missing

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/Upgrade.ini]: oops, done
good eye

**Comment by xezon** [comment]
All files added. Not tested in game.

**Comment by commy2** [comment]
Tested, werks. 👍

**Comment by ExiLe-ZH** [comment]
> Tested, werks. 👍

Always good, working twerks :D

**Pull request closed without merging**